src/Symfony/Component/Lock/CHANGELOG.md

```diff
@@ -5,3 +5,4 @@ CHANGELOG
 -----
 
  * added the component
+ * deprecated the `$blocking` argument of the `SemaphoreStore::isSupported()` method
```

Comment:
The component is not released yet. Does it make sense to deprecate think here?

Reply:
It's usefull to see deprecations linked to 3.4 here I think, else no textual informations is written elsewhere.

Reply:
Lock was not part of 3.3, so there is nothing to deprecate.

Reply:
Ok, line removed.